apps/frontend/src/app/3_organisms/GettingStartedPopup/GettingStartedPopup.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { reactLocalStorage } from 'reactjs-localstorage';
+
+import {
+  Button,
+  ButtonType,
+  Checkbox,
+  Dialog,
+  Heading,
+  HeadingType,
+  Link,
+  Pagination,
+} from '@sovryn/ui';
+
+import { useIsMobile } from '../../../hooks/useIsMobile';
+import { translations } from '../../../locales/i18n';
+import { sovrynWikiLinks } from '../../../utils/constants';
+import styles from './GettingStartedPopup.module.css';
+
+const DEFAULT_PAGE_SIZE = 1;
```

Comment:
```suggestion
const pageSize = 1;
```
You don't offer any option to override `pageSize` so the naming is a bit misleading. Also, it's not a global constant but a local one so I think that camelCase should be fine (but it can be `PAGE_SIZE` if you agree on that with @soulBit).